examples/research_projects/wav2vec2/run_common_voice.py

```diff
@@ -391,11 +394,18 @@ def extract_all_chars(batch):
     # Preprocessing the datasets.
     # We need to read the aduio files as arrays and tokenize the targets.
     def speech_file_to_array_fn(batch):
-        speech_array, sampling_rate = torchaudio.load(batch["path"])
-        batch["speech"] = resampler(speech_array).squeeze().numpy()
+        try:
+            speech_array, sampling_rate = sf.read(batch["path"] + ".wav")
```

Comment:
is `sf.read` faster in your experiments? Also Common Voice is stored as `.mp3` I think -> don't think `soundfile` can read this no?

Reply:
Yes, in my Benchmarks soundfile is the fastest method to load audio, but only wav.
That's why I store the audio as wav after resampling